home.css

```diff
@@ -0,0 +1,66 @@
+  #banner {
+    background-image: linear-gradient(120deg, #91C4FF 0%, #F0F7FF 87%);
+    position: relative;
+    width: 100%;
+    height: 800px;
+  }
+  
+  #cloud-scroll {
+    background: url(http://9a2e1a2047765d2a004e-e6d2db425d8d4f2619e5cee56a48bcef.r55.cf2.rackcdn.com/clouds.png) repeat-x;
+    top: 0;
+    left: 0;
+    width: 100%;
+    height: 100%;
+    background-size: cover;
+    position: absolute;
+    -webkit-animation: 900000s backgroundScroll infinite linear;
+    -moz-animation: 900000s backgroundScroll infinite linear;
+    -o-animation: 900000s backgroundScroll infinite linear;
+    -ms-animation: 900000s backgroundScroll infinite linear;
+    animation: 900000s backgroundScroll infinite linear;
+    animation-iteration-count: infinite;
+    animation-delay: 0s;
+  }
+  
+  @-webkit-keyframes backgroundScroll {
+    from {
+      background-position: 0 0;
+    }
+    to {
+      background-position: -99999999px 0;
+    }
+  }
+  
+  @keyframes backgroundScroll {
+    from {
+      background-position: 0 0;
+    }
+    to {
+      background-position: -99999999px 0;
+    }
+  }
+.heading
+{
+  margin-top: 0;
+  padding: 10px;
+  position: absolute;
+  left: -100%;
+  width: 100%;
+  height: auto;
+  -webkit-animation: slide 0.5s forwards;
+  -webkit-animation-delay: 2s;
+  animation: slide 0.5s forwards;
+  animation-delay: 1s;
+}
+
+@-webkit-keyframes slide {
+    0% { left: -100%; }
+}
+
+@-webkit-keyframes slide {
+    100% { left: 0%; }
+}
+
+@keyframes slide {
+    100% { left: 0; }
+}
```

Comment:
There should be something for `slide` when it is `0%`. The other two are vendor prefixes. Also, I feel that one should be consistent with the units. Either stick to 0 or 0%.